languages/go/ast/ast_go.ml

```diff
@@ -93,6 +93,7 @@ and func_type = {
 and parameter_binding =
   | ParamClassic of parameter
   (* sgrep-ext: *)
+  | ParamMetavarEllipsis of ident
```

Comment:
You could add a comment here like 
"for metavar ellipsis identifier (e.g., $...ARGS), we usually don't have a special construct in the AST for it;
Id of ident can represent such a metavar. But for Go parameters, a single identifier is usually interpreted
as a type, so we can't reuse parameter.pname to store $...ARGS, hence this special construct".